src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -491,18 +491,6 @@ public static bool IsGrainMethodInvokerType(Type type)
             return generalType.IsAssignableFrom(type) && TypeHasAttribute(type, typeof(MethodInvokerAttribute));
         }
 
-#if NETSTANDARD_TODO
-        public static Type ResolveType(string fullName)
-        {
-            return Type.GetType(fullName, true);
-        }
-
-        public static bool TryResolveType(string fullName, out Type type)
-        {
-            type = Type.GetType(fullName, false);
-            return type != null;
-        }
-#else
```

Comment:
This is a non-trivial change. I'd like to understand a little bit more of what prompted it, because this wouldn't just affect tests, but runtime behavior.